models/appModel.js

```diff
@@ -0,0 +1,40 @@
+var AppModel = Backbone.Model.extend({
```

Comment:
Kudos for having your model be the place where your data is manipulated, but see below comments re: leveraging the framework.